setup.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from setuptools import find_packages, setup
+
+from django_oikotie import __version__
+
+with open(os.path.join(os.path.dirname(__file__), "README.md")) as readme:
+    README = readme.read()
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name="django_oikotie",
+    version=__version__,
+    packages=find_packages(exclude=["tests", "tests.*"]),
+    include_package_data=True,
+    license="MIT License",
+    description="Django app for Oikotie integration.",
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/City-of-Helsinki/django-oikotie",
+    author="City of Helsinki",
+    author_email="dev@hel.fi",
+    install_requires=[
+        "Django",
+        "lxml",
+    ],
+    setup_requires=["pytest-runner"],
+    tests_require=["pytest", "pytest-django"],
+    extras_require={
+        "graphql_api": [
+            "graphene>=2.0",
+            "graphene-django",
+            "graphql-core<3,>=2.1",
+            "graphql-relay<3,>=2",
+        ]
+    },
+    classifiers=[
+        "Environment :: Web Environment",
+        "Framework :: Django",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.7",
```

Comment:
I've been using python 3.8 for this project so far since that's the newest and I don't see a reason to use 3.7.